logic/src/commonTest/kotlin/com/wire/kalium/logic/sync/receiver/conversation/message/ReceiptMessageHandlerTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.wire.kalium.logic.sync.receiver.conversation.message
+
+import app.cash.turbine.test
+import com.wire.kalium.logic.data.conversation.ClientId
+import com.wire.kalium.logic.data.id.QualifiedID
+import com.wire.kalium.logic.data.message.Message
+import com.wire.kalium.logic.data.message.MessageContent
+import com.wire.kalium.logic.data.message.receipt.ReceiptRepository
+import com.wire.kalium.logic.data.message.receipt.ReceiptRepositoryImpl
+import com.wire.kalium.logic.data.message.receipt.ReceiptType
+import com.wire.kalium.logic.framework.TestConversation
+import com.wire.kalium.logic.framework.TestMessage
+import com.wire.kalium.logic.framework.TestUser
+import com.wire.kalium.logic.sync.receiver.message.ReceiptMessageHandler
+import com.wire.kalium.persistence.TestUserDatabase
+import com.wire.kalium.persistence.dao.ConversationIDEntity
+import com.wire.kalium.persistence.dao.UserIDEntity
+import kotlinx.coroutines.test.runTest
+import kotlinx.datetime.Clock
+import kotlinx.datetime.Instant
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+class ReceiptMessageHandlerTest {
```

Comment:
This suite is quite nice in the sense that it tests the integration from the handler all the way down to the Database and back through the repository.

```mermaid
graph TD
    TestSuite --> Handler
    Handler --> Repository
    Repository --> Database
    Database --> Repository
    Repository --> TestSuite
```